POTR POTS

mykola-koval opened the pull request:
[POTR POTS](https://mykola-koval.github.io/Potr_Pots)

Denchepornyuk reviewed (changes requested):
https://bottlenose-barometer-ccb.notion.site/Potr-Pots-Koval-Mykola-bfed3128b4c84b1fba4b6b892d04fec1

mykola-koval commented:
UPD 14/12:
-form rework: added required etc
-form no longer reloads the page
-form no longer breaks adaptiveness
- added hover effects to footer icons
- header and footer links fixed
- changes to background rectangle
- added animation to basket

Denchepornyuk commented:
![Screenshot 2021-12-14 at 08 46 23](https://user-images.githubusercontent.com/37452763/145946998-00a0f678-3a2d-4a89-8257-5b0d36a62741.png)
на широких екранах контент не має розпливатись по сторонах, тримай його по центру
Задній фон можна розтягувати на всю ширину

Denchepornyuk commented:
![Screenshot 2021-12-14 at 08 48 31](https://user-images.githubusercontent.com/37452763/145947240-554c8351-1aca-4d29-800f-d046d027ccad.png)
цей блок кращи тримати по центру на менших екранах

Denchepornyuk commented:
![Screenshot 2021-12-14 at 08 49 38](https://user-images.githubusercontent.com/37452763/145947380-07b47738-6dff-4bb5-bf1f-2d455caa9db2.png)
білого простору з правої частини немає бути, в дизайні такого не знайшов

Denchepornyuk commented:
![Screenshot 2021-12-14 at 08 50 26](https://user-images.githubusercontent.com/37452763/145947469-df457f28-8b6d-4440-84e5-c0c402c12ebf.png)
на iphone 5 / se є біла полоска з правого боку

Denchepornyuk commented:
<img width="749" alt="Screenshot 2021-12-17 at 12 28 49" src="https://user-images.githubusercontent.com/37452763/146530629-cc82eca2-9f7a-4068-a4f9-765e0930c7d6.png">
ширина екрану 800 текст ховається
